F/FLINT/build_tarballs.jl

```diff
@@ -16,12 +17,8 @@ cd $WORKSPACE/srcdir
 cd flint2/
 if [[ ${target} == *musl* ]]; then
    # because of some ordering issue with pthread.h and sched.h includes
-   export CFLAGS=-D_GNU_SOURCE=1
-   # and properly define _GNU_SOURCE here as well to avoid many warnings
-   sed -i -e 's/#define _GNU_SOURCE$/#define _GNU_SOURCE 1/' thread_pool.h configure
+   #export CFLAGS=-D_GNU_SOURCE
```

Comment:
I want to see what the exact errors are (resp. if they still are there). If so, we can think about submitting a "proper" fix (or else an issue report) upstream.